src/routes/+page.svelte

```diff
@@ -1,2 +1,157 @@
-<h1>Welcome to SvelteKit</h1>
-<p>Visit <a href="https://kit.svelte.dev">kit.svelte.dev</a> to read the documentation</p>
+<script>
+  import Button from '../lib/button.svelte';
+</script>
+
+<div>
```

Comment:
Maybe make these buttons centered with a row class?

Reply:
Resolved!  Added a global CSS class to help with this